apps/system/test/unit/updatable_test.js

```diff
@@ -299,6 +299,32 @@ suite('system/Updatable', function() {
         assert.isFalse(subject.downloading);
       });
     });
+
+    suite('signal system update ready', function() {
+      var addEventListenerSpy, dispatchEventSpy;
+
+      setup(function() {
+        asyncStorage.setItem(SystemUpdatable.KNOWN_UPDATE_FLAG, true);
+        dispatchEventSpy    = this.sinon.spy(SystemUpdatable.prototype,
+                                             '_dispatchEvent');
+        addEventListenerSpy = this.sinon.spy(window, 'addEventListener');
+        subject = new SystemUpdatable(42);
```

Comment:
ahh the magic 42 that does everything for SystemUpdatable. soon all this will be 42. it is the answer to everything